controllers/storagecluster/odfinfoconfig.go

```diff
@@ -218,9 +218,13 @@ func getStorageSystemName(storageCluster *ocsv1.StorageCluster) (string, error)
 
 }
 
-func getOcsVersion(r *StorageClusterReconciler, storageCluster *ocsv1.StorageCluster) (string, error) {
+func getOcsVersion(r *StorageClusterReconciler) (string, error) {
 	var csvs operatorsv1alpha1.ClusterServiceVersionList
-	err := r.Client.List(r.ctx, &csvs, client.InNamespace(storageCluster.Namespace))
+	operatorNamespace, err := util.GetOperatorNamespace()
```

Comment:
`OperatorNamespace` is already part of the `StorageClusterReconciler` please use it from there.